packages/compass-e2e-tests/helpers/commands/scroll-to-virtual-item.ts

```diff
@@ -80,25 +126,15 @@ export async function scrollToVirtualItem(
           scrollContainer.scrollTop = nextScrollTop;
         },
         containerSelector,
-        scrollTop
+        scrollTop,
+        // Due to interprocess, we can not pass a function here.
+        // So, we stringify it here and then eval to execute it
+        config.getScrollContainer.toString()
```

Comment:
It is a pity. I guess whatever magic they do to make browser.execute() work doesn't extend to parameters.